datahub-web-react/.eslintrc.js

```diff
@@ -19,29 +18,34 @@ module.exports = {
         project: './tsconfig.json',
     },
     rules: {
-        eqeqeq: ['error', 'always'],
-        'react/destructuring-assignment': 'off',
-        'no-console': 'off',
-        'no-debugger': 'warn',
-        'require-await': 'warn',
+        '@typescript-eslint/no-explicit-any': 'off',
+        'arrow-body-style': 'off',
+        'class-methods-use-this': 'off',
+        'import/no-extraneous-dependencies': 'off',
         'import/prefer-default-export': 'off', // TODO: remove this lint rule
-        'import/extensions': 'off',
-        'react/jsx-props-no-spreading': 'off',
+        'no-console': 'off',
         'no-plusplus': 'off',
         'no-prototype-builtins': 'off',
-        'react/require-default-props': 'off',
+        'no-restricted-exports': ['off', { restrictedNamedExports: ['default', 'then'] }],
         'no-underscore-dangle': 'off',
+        'no-unsafe-optional-chaining': 'off',
+        'prefer-exponentiation-operator': 'off',
+        'prefer-regex-literals': 'off',
+        'react/destructuring-assignment': 'off',
+        'react/function-component-definition': 'off',
+        'react/jsx-no-bind': 'off',
+        'react/jsx-no-constructed-context-values': 'off',
+        'react/jsx-no-useless-fragment': 'off',
+        'react/jsx-props-no-spreading': 'off',
+        'react/no-unstable-nested-components': 'off',
+        'react/require-default-props': 'off',
```

Comment:
so these lint changes were necessary as we bumped dependencies we had new linting rules being applied that we didn't want applied?

nothing I'm seeing is concerning to me just wanted to call it out

Reply:
Yeap! That's right 😃